.circleci/config.yml.jinja

```diff
@@ -385,12 +422,33 @@ jobs:
                 command: make libp2p_helper
             - run:
                   name: Run unit tests
-                  command: ./scripts/skip_if_only_frontend.sh bash -c 'source ~/.profile && dune build --profile={{profile}} -j8 && (dune runtest src/lib --profile={{profile}} -j8 || (./scripts/link-coredumps.sh && false))'
+                  command: ./scripts/skip_if_only_frontend.sh bash -c 'source ~/.profile && dune build --profile={{profile}} -j8 src/app/cli/src/coda.exe src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe && (dune runtest src/lib --profile={{profile}} -j8 || (./scripts/link-coredumps.sh && false))'
```

Comment:
Why do we need runtime_genesis_ledger here?

Reply:
One of the unit tests failed if I didn't build the genesis ledger executable.

Reply:
We used to run `dune b` to build all the code; that breaks here because the nonconsensus code won't build with `dev`; instead, we build executables to force needed libraries to build.